src/server/memory/usage.h

```diff
@@ -219,26 +227,68 @@ class ColdObjectTracker
       return true;
     }
 
-    Status UnRef(const ID& id) {
+    /**
+     * @brief Here we have two actions: 1. delete from lru_list
+     *        2. delete from spilled_obj_
+     * @param id is the objectID
+     * @param fast_delete indicates if we directly remove the spilled object
+     * without reload
+     * @param store_ptr is used for spill
+     * @return * Status
+     */
+    Status Unref(const ID& id, bool fast_delete,
+                 std::shared_ptr<Der> store_ptr) {
       std::unique_lock<decltype(mu_)> locked;
       auto it = map_.find(id);
       if (it == map_.end()) {
+        auto it = spilled_obj_.find(id);
+        if (it == spilled_obj_.end()) {
+          return Status::OK();
+        }
+        if (!fast_delete) {
+          RETURN_ON_ERROR(store_ptr->ReloadPayload_(id, it->second));
+        }
```

Comment:
What will happen, if fast_delete is true?

Reply:
`fast_delete` is used to denote the situation that, if the user called `delete`-liked rpc/ipc to the server, then we shouldn't reload the file from disk, instead we should directly remove the payload.

Reply:
When I want to add the delete file logic in [here](https://github.com/v6d-io/v6d/pull/815/commits/3439b400171306cc7de3cab0a8c52ecfa1dc2632#diff-d7ef7d2035b2b3d576bc1a4f04bf065eeab33c3f8baf4c9fcb3e0f55c6d1eec6R114), the program complains for double free